docs/application/tizen-studio/troubleshooting/troubleshooting.md

```diff
@@ -0,0 +1,230 @@
+2. On **General tab** in the Emulator configuration window, set **OpenGL ES version to version 2.0** or to **version 3.0.**
+
+## Issue: UI display perspective issues on macOS
+**Description:**: After updating Tizen Studio, UI perspectives are not displayed correctly.
+
+**Resolution:**
+
+- In macOS&reg;, if the UI perspectives are not displayed after you update Tizen Studio, it is recommended to restart the IDE, doing so clears the cache from the previous installation.
+
+- In case you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+
+## Issue: Cannot view Connection Explorer
+
+**Description:** Not able to view Connection Explorer.  
+
+**Resolution:**
+
+Since Tizen Studio 2.0,**Connection Explorer** is replaced with **Device Manager**. Additionally, you can fix the issue as follows:
+
+- Reset the perspective:
+  1. In the Tizen Studio menu, select **Window > Perspective > Reset Perspective**.
+  2. After updating Tizen Studio to a higher version, run the following command:
+  `eclipse.exe -clean -clearPersistedState` 
+  3. Launch Tizen Studio.
+
+> [!NOTE]
+> You can create unit tests for Tizen Studio versions higher than 2.3.1. 
+
+## Issue: Preview tab does not appear
+
+**Description:**
+
+The preview tab in the Web Page Editor does not appear. 
+
+**Resolution:**
+- Use Web SDK HTML Editor that has enhanced features compared to the Web Page Editor. 
+- Instead of the Preview tab in the Web Page Editor, use the preview feature by pressing the **Ctrl + 4** combination of keys in Web SDK HTML Editor.
+- In Rapid Development Support (RDS) mode, check for the web unit test results update status.
+
+## Issue: Cannot launch Tizen Studio with GDB Client
+
+**Description:** Tizen Studio fails to launch with GDB client on Ubuntu&reg; 18.04.
+**Resolution:**
+
+Ubuntu&reg; 18.04 does not come with `readline6` dev package. To install the `readline6` dev package, run the following command (for 64 bit system):
+
+> [!NOTE]
+> Xenial (16.04) contains a `libreadline6-dev` package, but Ubuntu&reg; 18.04 does not. You can download and install the package with the following commands:
+
+```
+$ wget http://mirrors.edge.kernel.org/ubuntu/pool/main/r/readline6/libreadline6-dev_6.3-8ubuntu2_amd64.deb
+$ wget http://mirrors.edge.kernel.org/ubuntu/pool/main/r/readline6/libreadline6_6.3-8ubuntu2_amd64.deb 
+$ sudo dpkg -i libreadline6_6.3-8ubuntu2_amd64.deb
+$ sudo dpkg -i libreadline6-dev_6.3-8ubuntu2_amd64.deb 
+```
+
+## Issue: Emulator stuck at booting kernel
+
+**Description:** 
+
+Emulator gets stuck at booting kernel 
+
+**Resolution:**
+
+ The cause of such an issue is that the user has insufficient permissions. To resolve this problem, after installing KVM for Ubuntu&reg; 18.04, run the following command:
+
+```
+$ sudo adduser $USER kvm
+```
+After the command is executed, "kvm" group (/dev/kvm) gets added into the group file (/etc/group),. 
```

Comment:
Please remove the comma at the end of the sentence

Reply:
Done